compiler/testData/codegen/box/inlineClasses/kt44867.kt

```diff
@@ -0,0 +1,9 @@
+// WITH_RUNTIME
+// IGNORE_BACKEND: JVM
+
+open class BaseWrapper<T>(val response: T)
+class Wrapper(result: Result<String>) : BaseWrapper<Result<String>>(result)
+
+fun box(): String {
+    return Wrapper(Result.success("OK")).response.getOrThrow())
```

Comment:
Looks like there's an extra `)` here, which is why this test fails for example for JS, JS_IR and WASM. I wonder why it passes for JVM_IR though...

Reply:
Fixed. That's truly odd, though.